src/processes.py

```diff
@@ -0,0 +1,430 @@
+              " evaluation mode.")
+        for idx, eval_mode in enumerate(EvaluationMode):
+            print(f"{idx}:\t{eval_mode}")
+        print("Write 'exit' to quit.")
+
+        # Handle evaluation mode input
+        while True:
+            try:
+                idx = input("> ")
+                if "exit" == idx:
+                    quit = True
+                    break
+                elif int(idx) >= 0 and int(idx) < len(EvaluationMode):
+                    picked_eval = eval_modes[int(idx)]
+                    break
+            except ValueError:
+                continue
+
+        if quit:
+            break
+
+        #####################
+        # Presentation mode #
+        #####################
+        # Input prompt
+        print("Pick presentation mode by writing the index of the wanted"
+              " presentation mode.")
+        for idx, pres_mode in enumerate(PresentationMode):
+            print(f"{idx}:\t{pres_mode}")
+        print("Write 'all' to present all learning profiles.")
+        print("Write 'exit' to quit.")
+
+        # Handle presentation mode input
+        while True:
+            try:
+                idx = input("> ")
+                if "exit" == idx:
+                    quit = True
+                    break
+                elif "all" == idx:
+                    lps_present = lps_desc
+                    presentation_phase(learning_profiles=lps_present,
+                                       eval=picked_eval,
+                                       nr_models=n_models)
+                    break
+                elif int(idx) >= 0 and int(idx) < len(PresentationMode):
+
+                    # Set nr of models per category item
+                    n_models_per_category_item = None
+                    while True:
+                        print("Write the number of models per category item to"
+                              " use. Where the categories are AL/ML/DS, "
+                              "and category items is a specific "
+                              "AL/ML/DS method "
+                              "(-1 means all models)")
+                        n = int(input("> "))
+                        if n >= -1:
+                            n_models_per_category_item = n
+                            break
+
+                    # Filter learning profiles given the arguments
+                    lps_present = _get_sorted_specific_learning_profiles(
+                        lps_desc, picked_eval, pres_modes[int(idx)],
+                        n_models_per_category_item)
+                    print(f"{len(lps_present)} out of {len(lps_desc)} Learning"
+                          " profiles will be used in the plot.")
+
+                    # Run presentation phase to plot the results
+                    presentation_phase(learning_profiles=lps_present,
+                                       eval=picked_eval,
+                                       nr_models=n_models)
```

Comment:
What is the difference between n_models and n_models_per_category_item ? 
Only n_models seems to do anything. 
If you specify 2 models at launch then write -1 in the prompt later, the print says all 384 models will be presented, but only 2 shows in the plot.